src/routes/news/fetchMoreData.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import type { Dispatch } from 'redux'
+import { Endpoint, Payload } from '@integreat-app/integreat-api-client'
+import startFetchMoreAction from './actions/startFetchMoreAction'
+import finishFetchMoreAction from './actions/finishFetchMoreAction'
+import type { StoreActionType } from '../../modules/app/StoreActionType'
+
+async function fetchData<P, T> (
```

Comment:
Is this different from the actual `fetchData`? Or is the only difference what actions are dispatched? If so, we could maybe also just use a flag for this and use the actual `fetchData`.

Reply:
I thought about passing the actions as a parameter instead of making it global but I wasn't sure about it that's why I created a new function. I think it would be cleaner than using a flag if you ever needed to dispatch other actions
What do you think? @klinzo  @maxammann

Reply:
We could pass two functions as optional params which map endpoint and payload to start/finish action. But I'm also not sure. @maxammann what do you think?

Reply:
Still relevant

Reply:
tbh. I'm not sure what to do here? Have you talked to @maxammann about this?

Reply:
Yeah sure, sorry. We have to wait for an answer here. My comment was just meant as a hint for myself and @maxammann